keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone_test.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import tensorflow as tf
+from absl.testing import parameterized
+from tensorflow import keras
+
+from keras_cv.models.backbones.efficientnet_v2.efficientnet_v2_backbone import (
+    EfficientNetV2Backbone,
+)
+from keras_cv.models.backbones.efficientnet_v2.efficientnet_v2_backbone import (
+    EfficientNetV2SBackbone,
+)
+from keras_cv.utils.train import get_feature_extractor
+
+
+class EfficientNetV2BackboneTest(tf.test.TestCase, parameterized.TestCase):
+    def setUp(self):
+        self.input_batch = tf.ones(shape=(8, 224, 224, 3))
+
+    def test_valid_call(self):
+        model = EfficientNetV2Backbone(
```

Comment:
Can we just load from a Preset here instead?

Referring to @ianstenbit 's [comment](https://github.com/keras-team/keras-cv/pull/1641/commits/699ff06fd421642e56e885c7eee0fcf7c84f3f2f#r1171871160) on #1641

Reply:
we can; but this way we get lots of coverage